state/spaces.go

```diff
@@ -284,7 +285,12 @@ func (st *State) SpaceByName(name string) (*Space, error) {
 	return &Space{st, doc}, nil
 }
 
-// AllSpaceInfos return SpaceInfos for all spaces in the model.
+// AllSpaceInfos returns SpaceInfos for all spaces in the model.
+// TODO (manadart 2020-04-09): Instead of calling NetworkSpace here,
```

Comment:
This is an important one to address soon.
Unwitting callers of this method are hitting the DB more than they may think.
I have ideas about a factory and separate state-to-core mapping separation around this.